v2-migration/src/test/java/software/amazon/awssdk/v2migration/ChangeSdkTypeTest.java

```diff
@@ -216,30 +215,4 @@ void shouldChangeFieldsInInnerClass() {
             )
         );
     }
-
-    @Test
-    @EnabledOnJre({JRE.JAVA_8})
-    void hasUnsupportedFeature_shouldSkip() {
```

Comment:
Can we still keep the test? It should still pass, right?

Reply:
Yup, adding back test, with Lambda invoke class